src/main/java/com/courtcanva/ccfranchise/dtos/OpenOrderResponseDto.java

```diff
@@ -0,0 +1,43 @@
+package com.courtcanva.ccfranchise.dtos;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import java.math.BigDecimal;
+import java.time.OffsetDateTime;
+import java.util.Objects;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class OpenOrderResponseDto {
+
+    private OffsetDateTime createdTime;
+
+    private String customerId;
+
+    private String contactInformation;
+    // TODO: <future function> get customerName, address, phone no. from final database (cc-admin repository).
+
+    private String postcode;
+
+    private BigDecimal totalAmount;
+
+    @Override public boolean equals(Object o) {
```

Comment:
why we need override equals and hashCode?

Reply:
The reason is, in `OrderServiceTest.getOpenOrdersByFranchiseeIdTest()`, I used `assertEquals()` to validate `OrderService.getOpenOrdersByFranchiseeId(id)` returns **expected dto**. 

So the Dto class's equals() and hashCode() need to be override.

Reply:
should not do this way only because you need it in the test. To test correctly, you can use different ways to compare